scripts/pi-hole/php/FTL.php

```diff
@@ -8,12 +8,12 @@
 
 $piholeFTLConfFile = "/etc/pihole/pihole-FTL.conf";
 
-function piholeFTLConfig()
+function piholeFTLConfig($force=false)
```

Comment:
I'm not sure if I understand what you are doing here with the `force` flag. What is your goal?

Reply:
This function caches the result from reading `pihole-FTL.conf` on first read and returns the cached value on any subsequent call without re-reading.

This is a problem with the new code: We read `pihole-FTL.conf` in `savesettings.php` when saving the settings (old values), then the script calls `pihole -a rataelimit x y`. Later on, we call `piholeFTLConfig()` again but this still contains the *old* values as the function returns from cache. This new option allows us to force re-reading instead of using the cache. This fixes the issue. The `force` flag defaulting to `false` preserves the current behavior in all other places where we don't need this.

Reply:
Thanks. 

So I guess as a side effect this just fixed https://github.com/pi-hole/AdminLTE/issues/1783 as well?

Reply:
Oh yes, I didn't even see this other issue.